src/app/hooks/client/rpc/queries/useBbnQuery.ts

```diff
@@ -38,8 +41,33 @@ export const useBbnQuery = () => {
   const rewardsQuery = useClientQuery({
     queryKey: [BBN_REWARDS_KEY, bech32Address],
     queryFn: async () => {
+      if (isE2ETest) {
```

Comment:
At the moment trying to figure out a better way for mocking the reward gauge response